packages/now-next/src/index.ts

```diff
@@ -975,19 +998,6 @@ export const build = async ({
     Object.keys(prerenderManifest.lazyRoutes).forEach(route =>
       onPrerenderRoute(route, true)
     );
-
-    // Dynamic pages for lazy routes should be handled by the lambda flow.
-    Object.keys(prerenderManifest.lazyRoutes).forEach(lazyRoute => {
```

Comment:
We still need this for backwards compat don't we? When `.dataRoutes` doesn't exist.

Reply:
Right, re-added for backwards compatibility